packages/chrome/src/elements/Chrome.tsx

```diff
@@ -23,60 +23,55 @@ interface IChromeProps extends HTMLAttributes<HTMLDivElement> {
 /**
  * Accepts all `<div>` attributes and events
  */
-const Chrome = React.forwardRef<HTMLDivElement, IChromeProps & ThemeProps<DefaultTheme>>(
-  ({ hue, theme, isFluid, ...props }, ref) => {
-    const isLightMemoized = useMemo(() => {
-      if (hue) {
-        const backgroundColor = getColor(hue, 600, theme);
-        const LIGHT_COLOR = 'white';
+const Chrome = React.forwardRef<HTMLDivElement, IChromeProps>(({ hue, isFluid, ...props }, ref) => {
+  const theme = useContext(ThemeContext);
```

Comment:
The `withTheme` usage confuses the Storybook story-source plugin. This aligns the Theme usage with newer components.

Reply:
@austin94 does this mean we'll be subsequently dropping `withTheme` throughout?

Reply:
I think it would be a good idea. Cleans up our prop-sheets throughout and also aligns us with eventual changes in `styled-components` v6+